script.js

```diff
@@ -7,3 +7,50 @@ function mobileMenu() {
+    technologies: ['html', 'css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    seeLive: 'https://nedramrez.github.io/portfolio/',
+    seeSource: 'https://github.com/nedramrez/portfolio',
+  },
+  thirdCard: {
+    name: 'Tonic',
+    image: '.assets/desktop-snapshoot-1.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    technologies: ['html', 'css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    seeLive: 'https://nedramrez.github.io/portfolio/',
+    seeSource: 'https://github.com/nedramrez/portfolio',
+  },
+  fourthCard: {
+    name: 'Tonic',
+    image: '.assets/desktop-snapshoot-1.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    technologies: ['html', 'css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    seeLive: 'https://nedramrez.github.io/portfolio/',
+    seeSource: 'https://github.com/nedramrez/portfolio',
+  },
+};
```

Comment:
- [ ] Kindly consider making the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads).
- [ ] Also consider modal popup data is populated with the data from the projects array.